htdocs/core/modules/bom/doc/doc_generic_bom_odt.modules.php

```diff
@@ -296,7 +296,7 @@ public function write_file($object, $outputlangs, $srctemplatepath, $hidedetails
 				$contactobject = null;
 				if (!empty($usecontact)) {
 					// On peut utiliser le nom de la societe du contact
-					if ($usecontact && ($object->contact->fk_soc != $object->thirdparty->id && (!isset($conf->global->MAIN_USE_COMPANY_NAME_OF_CONTACT) || !empty($conf->global->MAIN_USE_COMPANY_NAME_OF_CONTACT)))) {
+					if ($usecontact && ($object->contact->fk_soc != $object->thirdparty->id && !empty($conf->global->MAIN_USE_COMPANY_NAME_OF_CONTACT))) {
```

Comment:
If MAIN_USE_COMPANY_NAME_OF_CONTACT is not set, we want the same behaviour than having the value set to 1. So old code seems good to me.

Reply:
Oh, I see, but how can I disable that option then?
If I set MAIN_USE_COMPANY_NAME_OF_CONTACT to an empty value, it is deleted immediately, thus it is not set anymore and the option is active. If I set it to something non-empty, the option is active, too. So effectively, it is always active...

Reply:
You can set the parameter to '0' into home - setup - other

Reply:
Yeah, sure, but then the variable is not empty but has the value `"0"`, hasn't it?

Reply:
Ah, I just checked the manual, I wasn't aware that PHP considers `"0"` the same as `false` and thus empty... Sorry for the noise then.